index.html

```diff
@@ -3,23 +3,8 @@
   
   <head>
     <title>Amiso-web Design</title>
-    <link rel="stylesheet" href="header.css" />
-    <link rel="stylesheet" href="id.css" />
-    <link rel="stylesheet" href="amiso-header.css" />
-    <link rel="stylesheet" href="amiso-body.css" />
-    <link rel="stylesheet" href="h.css" />
-    <link rel="stylesheet" href="faq.css" />
-    <link rel="stylesheet" href="pie.css" />
-    <link rel="stylesheet" href="testimonials.css" />
-    <link rel="stylesheet" href="web-design-grid.css" />
-    <link rel="stylesheet" href="works-completed.css" />
-    <link rel="stylesheet" href="company-benefits.css" />
-    <link rel="stylesheet" href="news-articles.css" />
-    <link rel="stylesheet" href="contact-now.css" />
-    <link rel="stylesheet" href="subscribe.css" />
-    <link rel="stylesheet" href="experienced-designers.css" />
-    <link rel="preconnect" href="https://fonts.googleapis.com">
-    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link rel="stylesheet" href="main.css" />

```

Comment:
This file would now contain all the CSS. Ensure to merge the rest of the CSS files into this single file